plugins/services/src/js/service-configuration/PodEnvironmentVariablesConfigSection.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapTable from '../components/ConfigurationMapTable';
+import ConfigurationVisible from '../components/ConfigurationVisible';
+import ServiceConfigDisplayUtil from '../utils/ServiceConfigDisplayUtil';
+
+module.exports = ({appConfig}) => {
+  let envVars = Object.keys(appConfig.environment || {}).reduce((memo, key) => {
```

Comment:
Let's use default assignment instead of cramming more into one function:
`module.exports = ({appConfig: environment = {}, containers = []}) => {`
or:
```js
module.exports = ({appConfig}) => {
  let {environment = {}, containers = []} = appConfig;
  ...
```